teuthology/task/install/rpm.py

```diff
@@ -111,11 +111,11 @@ def _update_package_list_and_install(ctx, remote, rpm, config):
     """
     remote_os = remote.os
     rpm = _package_overrides(rpm, remote_os)
-    log.info("Installing packages: {pkglist} on remote rpm {arch}".format(
-        pkglist=", ".join(rpm), arch=remote.arch))
 
     dist_release = remote_os.name
     repos = config.get('repos')
+    install_packages = config.get('install_packages', True)
```

Comment:
@kshtsk If you change `True` to `False` here, the install task will only add repos (which we want) but not try to install packages. This will avoid the problem we are having with ceph-test RPM not installing.